src/core/pointcloud/qgspointcloudstatistics.cpp

```diff
@@ -111,3 +127,54 @@ void QgsPointCloudStatistics::combineWith( const QgsPointCloudStatistics &stats
   }
   mSampledPointsCount += stats.mSampledPointsCount;
 }
+
+QJsonObject QgsPointCloudStatistics::toJson() const
+{
+  QJsonObject obj;
+  obj.insert( QStringLiteral( "sampled-points" ), QJsonValue::fromVariant( sampledPointsCount() ) );
+  QJsonObject stats;
+  for ( QString &attr : mStatisticsMap.keys() )
+  {
+    QgsPointCloudStatistics::AttributeStatistics stat = mStatisticsMap.value( attr );
+    stats.insert( attr, QgsPointCloudStatistics::fromStatsToJson( stat ) );
+  }
+  obj.insert( QStringLiteral( "stats" ), stats );
+  return obj;
+}
+
+QJsonObject QgsPointCloudStatistics::fromStatsToJson( QgsPointCloudStatistics::AttributeStatistics &stats )
+{
+  QJsonObject obj;
+  obj.insert( QStringLiteral( "minimum" ), stats.minimum );
+  obj.insert( QStringLiteral( "maximum" ), stats.maximum );
+  obj.insert( QStringLiteral( "mean" ), stats.mean );
+  if ( !std::isnan( stats.stDev ) )
+  {
+    obj.insert( QStringLiteral( "stDev" ), stats.stDev );
+  }
+  obj.insert( QStringLiteral( "count" ), stats.count );
```

Comment:
why do we store/read/calculate "count" for each attribute? isn't it always equal to the number of (sampled) points?

Reply:
Without the count in AttributeStatistics I don't think we can accumulate means or standard deviation. We can choose to not store it at all and use the samples count instead, what do you think?